doc/content/integrations/cloud-integrations/widgelix/_index.md

```diff
@@ -9,68 +9,50 @@ aliases:
 
 <!--more-->
 
 ## Prerequisites
 
-1. A [user account on Widgelix](https://widgelix.com/register).
+1. A [user account on Widgelix](https://app.widgelix.com/auth/register).
 
 ## Setup Widgelix
 
-### Create a Data Converter
+### Create a device template from Repository
 
-{{< note >}} Skip this section if you are already decoding data on {{% tts %}} using [payload formatters]({{< ref "/integrations/payload-formatters" >}}). {{</ note >}}
+Navigate to **Device Types** on the left hand menu, then click the **Load From Repository** button.
 
-Navigate to **Data Converters** on the left hand menu, then click **Create Data Converter**.
-Give a **Name** to your converter, then edit the default JavaScript converter code or create a new one.
+Select required device and give a **Name** to your template and click **Save** button.
 
-Optionally, you may provide a test payload and check the converter. The resulting output of your converter should be a flat JSON object.
+{{< figure src="add-device-type.png" alt="Add device template" >}}
 
-{{< figure src="data-converter.png" alt="Create Data Converter" >}}
-
-### Create Device Type
-
-Device types allow to join same devices in a single group, manage simultaneously and assign **Rules**, **Widget Sets**, **Data Converters** and **Data Flows**.
-
-Navigate to **Device Types** on the left hand menu, then click **Create Device Type** button.
-
-Give a **Name** to your device type, and fill the **Description** and **Manufacturer** fields (optional).
-
-If you previously [created a data converter]({{< ref "/integrations/cloud-integrations/widgelix#create-a-data-converter" >}}), you can assign it under **Uplink Data Converter**.
-In the **Data Fields** input, you need to provide all required keys from your resulting flat JSON object.
-
-{{< figure src="device-type.png" alt="Create Device Type" >}}
-
-### Add Device
+### Add new device based on the template
 
 Navigate to **Devices** on the left hand menu, then click the **Create Device** button.
 
-Give a **Name** to your device and provide its **DevEUI**. Assign the previously created **Device Type**.
-
-{{< figure src="create-device.png" alt="Create Device" >}}
+Give a **Name** to your device, type **DevEUI**, assign previously created **Device Type** template and click **Save** button.
 
-### Create Data Flow
+{{< figure src="add-device.png" alt="Add device" >}}
 
-Navigate to **Data Flows** on the left hand menu, then click the **Create New Data Flow** button.
+### Configure Connectivity
 
-Give a **Name** to your data flow, select **LoRaWAN (TTN)** connectivity and assign the previously created **Device Type**.
+Navigate to **Devices** on the left hand menu, then click on the created device.
 
-{{< figure src="create-data-flow.png" alt="Create Data Flow" >}}
+Go to **Settings** tab and click on **Connectivity** section. Then select {{% tts %}}, switch **Secure** option. And finally, copy given API Key.
```

Comment:
```suggestion
Go to **Settings** tab and click on **Connectivity** section. Then select {{% tts %}}, switch **Secure** option. And finally, copy the given API Key because you'll need it in the next step.
```